fhir-resource-tooling/src/main/java/com/ibm/cohort/tooling/fhir/ValueSetImporter.java

```diff
@@ -50,10 +63,19 @@
 
 		@Parameter(names = {"-h", "--help"}, description = "Show this help", help = true)
 		boolean isDisplayHelp;
-		
-		@Parameter(names = {"-p", "--file-system-output-path"}, description = "If specified, value sets will be converted to FHIR format and writen out to the file system path specified by this parameter. Value sets will not be imported into the FHIR server if this option is set.", required = false)
+
+		@Parameter(names = {"--output-locations"}, description = "If any but \"NONE\" is specified, value sets will be converted to FHIR format and written to a location specified by the bucket or filePath parameters. Value sets will not be imported into the FHIR server if this option is set. Options are COS, LOCAL, BOTH, NONE.")
+		String fileOutputLocation = "NONE";
```

Comment:
I think using an `Enum` here might work out better for you.
→ [EnumConverter.java](https://github.com/cbeust/jcommander/blob/master/src/main/java/com/beust/jcommander/converters/EnumConverter.java)

I believe that the default `--help` also includes the list of `Enum` options for you, so you wouldn't have to maintain it in the `description` field.

Reply:
Done!